tests/integration/test_token_provider.py

```diff
@@ -0,0 +1,24 @@
+from pretix_eth.providers import BlockscoutTokenProvider
+
+from eth_utils import (
+    is_boolean,
+    is_checksum_address,
+    is_bytes,
+    is_integer,
+)
+
+
+POA_DAI_TXN_HASH = '0x67c4d34ff58351d196bdd74160b3a93d4e3298c5dd8423aa954f268f3f54d610'
```

Comment:
When you update to them mainnet one you'll probably have to find a new address that has some existing DAI transfers.